docker/compose/.env

```diff
@@ -0,0 +1,3 @@
+ISOLATION_ID=latest
+REPO_VERSION=0.3.7-dev
```

Comment:
Is this the right version?

Reply:
o i guess it does match the .env file, I though it would be 0.4.1

Reply:
Something definitely went wrong there. Fixed now.